app/views/users/new.html.erb

```diff
@@ -1,23 +1,8 @@
 <% provide :title, t(".title") %>
+<% provide :button_text, t(".btn_create") %>
 <h1><%= t ".title" %></h1>
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
-    <%= form_for @user, url: signup_path do |f| %>
-      <%= render "shared/error_messages" %>
-
-      <%= f.label :name %>
-      <%= f.text_field :name, class: "form-control" %>
-
-      <%= f.label :email %>
-      <%= f.email_field :email, class: "form-control" %>
-
-      <%= f.label :password %>
-      <%= f.password_field :password, class: "form-control" %>
-
-      <%= f.label :password_confirm, t(".confirmation") %>
-      <%= f.password_field :password_confirm, class: "form-control" %>
-
-      <%= f.submit t(".btn_create"), class: "btn btn-primary" %>
-    <% end %>
+    <%= render "form" %>
```

Comment:
trong trường hợp này sẽ redirect trang login với alert là Please log in.
Anh có thể giải thích vì sao khi reload trình duyệt thì lại load /users/index không ạ?